src/languages/leaf.js

```diff
@@ -26,22 +26,57 @@ export default function(hljs) {
           {
             className: 'params',
             begin: '\\(',
-            end: '\\)',
+            end: '\\)\\:?',
             endsParent: true,
+            relevance: 7,
             contains: [
               {
                 className: 'string',
                 begin: '"',
                 end: '"'
               },
+              {
+                className: 'keyword',
+                begin: 'true|false|in',
+              },
               {
                 className: 'variable',
                 begin: '[A-Za-z_][A-Za-z_0-9]*'
-              }
+              },
+              {
+                className: 'operator',
```

Comment:
Operators can only exist within params?

Reply:
Correct. Leaf is a templating language so there's no use for them outside of a pair of parentheses